### DISCUSSION TITLE
how long to install on a macbook pro?

### DISCUSSION THREAD

**Discussion by danprater** [BLAST-WarpX/warpx#5437] [Installation]
Hi all, I'm just getting started with WarpX. 

The first installation command using conda, conda update -y -n base conda, is "Solving environment" now for, say, 20 minutes? is this normal? If so, cool. If not, what should I do?
Thank you!

**Comment by danprater** [comment]
it took about 4-5 hours but did complete

**Comment by RemiLehe** [comment]
Thanks for reporting this.
This is interesting. Note that the first installation command (`conda update -y -n base conda`) is not WarpX-specific. It does seem strange that it would take so long. In my case (on a MacBook Air), it took less than 5 min. Maybe your conda installation was somewhat old?
